projects/packages/forms/src/contact-form/css/file-field.css

```
@@ -273,19 +273,16 @@
 }
 
 .jetpack-form-file-field__progress-bar {
-	content:'';
+	content: '';
 	position: absolute;
 	top: 0;
 	bottom: 0;
 	left: 0;
 	right: 0;
 	border-radius: 50%;
 	background:
-		conic-gradient(
-			var( --jetpack--contact-form--text-color, #333 )
-			calc(var(--progress, 3) * 1%),
-			transparent calc(var(--progress, 3) * 1%)
-		);
+		conic-gradient(var(--jetpack--contact-form--text-color, #333)
+		calc(var(--progress, 3) * 1%), transparent calc(var(--progress, 3) * 1%));
```

Eew. Will it be happy with this instead?
```suggestion
		conic-gradient(var(--jetpack--contact-form--text-color, #333) calc(var(--progress, 3) * 1%), transparent calc(var(--progress, 3) * 1%));
```
I think it will, but if not even this would be better IMO
```suggestion
		conic-gradient(
			var(--jetpack--contact-form--text-color, #333) calc(var(--progress, 3) * 1%),
			transparent calc(var(--progress, 3) * 1%)
		);
```

---

I'd prefer the latter as well, but unfortunately neither of those work.

It doesn't complain about a full one-liner, but that too is ugly:
```
	background: conic-gradient(var(--jetpack--contact-form--text-color, #333) calc(var(--progress, 3) * 1%), transparent calc(var(--progress, 3) * 1%));
```

I was curious how it'd interact with the `@stylistic/max-line-length` rule (80 chars) that has yet to be enabled, too, but it turns out that rule ignores non-comments.

---

Perhaps we could override with "always-single-line"?
https://github.com/stylelint-stylistic/stylelint-stylistic/tree/main/lib/rules/declaration-colon-space-after#always-single-line

---

I'm fine with the one-liner that works. 🤷 There are various others like it (although maybe with less `var`) elsewhere in the codebase.

---

I explored overrides, and it resulted in a chain of overrides needed, so a one-liner it is: c38c4e66bdf